optimum/exporters/onnx/base.py

```diff
@@ -652,10 +652,15 @@ def add_past_key_values(self, inputs_or_outputs: Dict[str, Dict[int, str]], dire
         if direction not in ["inputs", "outputs"]:
             raise ValueError(f'direction must either be "inputs" or "outputs", but {direction} was given')
 
+        if direction == "inputs":
+            decoder_sequence_name = "past_sequence_length"
+        else:
+            decoder_sequence_name = "past_sequence_length + 1"
+
         name = "past_key_values" if direction == "inputs" else "present"
```

Comment:
Nit: the variable `name` could be instantiated in the new conditional block right above.

Reply:
Thank you!